pkg/apiserver/handlers.go

```diff
@@ -292,8 +292,11 @@ func (r *APIRequestInfoResolver) GetAPIRequestInfo(req *http.Request) (APIReques
 	// URL forms: /namespaces/{namespace}/{kind}/*, where parts are adjusted to be relative to kind
 	if currentParts[0] == "namespaces" {
 		if len(currentParts) < 3 {
-			requestInfo.Namespace = ""
 			requestInfo.Resource = "namespaces"
+			requestInfo.Namespace = ""
```

Comment:
Oh... I already did this in my pull.  But I think I did it a different way.

----- Original Message -----
> >  			requestInfo.Resource = "namespaces"
> > +			requestInfo.Namespace = ""
> 
> updated
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/GoogleCloudPlatform/kubernetes/pull/4342/files#r24533335